bin/test-ssh-key.py

```diff
@@ -7,48 +7,37 @@
 import sys
 import base64
 import struct
-from lepl.apps.rfc3696 import Email
 
 if len(sys.argv) != 2:
-  print "usage: " + sys.argv[0] + " <keystring>"
+  print("usage: " + sys.argv[0] + " <keystring>")
```

Comment:
The script is fine as is, but if you're interested, Python's `argparse` library is really useful for quickly checking CLI args and options. I usually set a docstring at the top of the file and then create the parser as a global, then invoke it near the end inside the typical `if __name__ == '__main__':` block:


```
"""
Verifies an SSH key for use in Ganeti's VPS console. (I think that's what this is...)
"""

parser = argparse.ArgumentParser(description=__doc__.strip())
parser.add_argument('key_file')

...

if __name__ == '__main__':
    args = parser.parse_args()  # automatically prints usage to stderr and exists on `-h` or invalid args
    verify_key(args.key_file)
```

Reply:
Send a PR!